freqtrade/rpc/rpc.py

```diff
@@ -105,22 +127,18 @@ def rpc_status_table(self) -> Tuple[bool, Any]:
             columns = ['ID', 'Pair', 'Since', 'Profit']
             df_statuses = DataFrame.from_records(trades_list, columns=columns)
             df_statuses = df_statuses.set_index(columns[0])
-            # The style used throughout is to return a tuple
-            # consisting of (error_occured?, result)
-            # Another approach would be to just return the
-            # result, or raise error
-            return False, df_statuses
+            return df_statuses
 
-    def rpc_daily_profit(
+    def _rpc_daily_profit(
             self, timescale: int,
-            stake_currency: str, fiat_display_currency: str) -> Tuple[bool, Any]:
+            stake_currency: str, fiat_display_currency: str) -> List[List[Any]]:
         today = datetime.utcnow().date()
         profit_days: Dict[date, Dict] = {}
 
         if not (isinstance(timescale, int) and timescale > 0):
-            return True, '*Daily [n]:* `must be an integer greater than 0`'
+            raise RPCException('*Daily [n]:* `must be an integer greater than 0`')
```

Comment:
i would move the markdown-formatting to each rpc implementation...

raising with `'must be an inteter greater than 0'` will allow more flexibility - it works fine with telegram - but will be a hurdle for rest/rpc/whatever implementations.

Reply:
Exactly my thoughts. I will do t hat in a follow up PR as this is already fairly bit.